form/config.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+//check if environment is local or prod
+if (stristr($_SERVER['HTTP_HOST'], 'local') || (substr($_SERVER['HTTP_HOST'], 0, 7) == '192.168')) {
+    //db connection
+    $dbHost = "localhost";
+    $dbName = "ballers_new";
+    $dbUser = "root";
+    $dbPass = "";
+    $mysqli = mysqli_connect($dbHost, $dbUser, $dbPass, $dbName);
+    
+    if (!$mysqli) {
+        echo "did  not connect";
+        echo "Error: Unable to connect to db" . PHP_EOL;
+        exit;
```

Comment:
Code Quality: Line 13 - 15 can be with `die “Error: Unable to connect to db" . PHP_EOL`

Reply:
this is why I didn't use `die` [https://www.php.net/manual/en/function.die.php](url)

Reply:
I agree with the first comment on the link. However, your implementation on lines 13 - 15 does the same thing as die (die is equivalent to exit() but you can include a custom error message) since you didn't handle the error in a way that will allow the users to continue using the application when they are unable to connect to the DB.